h2o-core/src/main/java/hex/grid/GridSearch.java

```diff
@@ -258,7 +272,8 @@ private void gridSearch(Grid<MP> grid) {
       grid.unlock(_job);
     }
   }
-
+  
+  private static final Set<String> IGNORED_FIELDS_PARAM_HASH = new HashSet(Arrays.asList("_export_checkpoints_dir"));
```

Comment:
you seem to want this constant, in which cas I would use Collections.immutableSet

Reply:
Good advice. Made it so.